templates/filebeat.yml.j2

```diff
@@ -104,6 +104,7 @@ output:
     # For Packetbeat, the default is set to packetbeat, for Topbeat
     # top topbeat and for Filebeat to filebeat.
     #index: filebeat
+{% endif %}
```

Comment:
The `{% if filebeat_ssl_certificate_file and filebeat_ssl_key_file %}` already applies above, so it wouldn't need to be defined a second time if Logstash isn't present... or am I reading the configuration wrong? The two top level keys are `elasticsearch` and `logstash`, and each has an independent `tls` configuration, and that level in the JSON object hierarchy would mean `tls` gets defined twice for `elasticsearch` if `filebeat_output_logstash_enabled` is false.

Reply:
OK. So additional "endif" has to be placed after "#curve_types: []  {% endif %}"